### PR TITLE
Use local storage for images (for now)

### PR DESCRIPTION
**Pull request by BrianBorge**
Use local (disk) storage for images (for now)
- This is easier than setting up and testing an AWS account
- The amount of images stored to start is small ( less than 50 )

### CODE REVIEW

**Review by alexandraborge — APPROVED** [review]

**Comment by alexandraborge** [comment]
Should we then add some different default avatars? Also, we should remove the button that allows you to add an image since the user won't be able to see their image on production

File: Gemfile [hunk 1]
```diff
@@ -15,14 +15,15 @@ gem 'image_processing', '1.9.3'
 gem 'mini_magick', '4.9.5'
 gem 'active_storage_validations', '0.8.2'
 gem "aws-sdk-s3", require: false
+gem 'pry-rails'
```
Comment (BrianBorge) [inline Gemfile:18]: This should probably exist in the dev/test group below

File: Gemfile [hunk 2]
```diff
@@ -15,14 +15,15 @@ gem 'image_processing', '1.9.3'
...
 
 group :development, :test do
   gem 'sqlite3', '~> 1.4'
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
 end
 
 group :production do
-  gem 'pg', '0.20.0'
+  gem 'pg'
```
Comment (BrianBorge) [inline Gemfile:26]: I would add the version back unless there was a reason it was removed

File: app/views/users/edit.html.erb [hunk 3]
```diff
@@ -12,9 +12,9 @@
     <%= f.text_field(:name) %><br>
     <%= f.label('Profile Photo') %><br>
     <%= image_tag(url_for(@user.display_image)) + '<br>'.html_safe if @user.avatar.present? %>
-    <%= f.file_field(:avatar) %><br>
+    <%#= f.file_field(:avatar) %><br>
```
Comment (BrianBorge) [inline app/views/users/edit.html.erb:15]: Can this erb comment be removed?

File: app/models/user.rb [hunk 4]
```diff
@@ -16,6 +16,12 @@ def display_image
 
   def default_values
     self.summary ||= 'I love holiday games! (Use this area to tell us about yourself and your top gift preferences. Be Specific!)'
-    self.avatar.attach(io: File.open(Rails.root.join('app', 'assets', 'images', 'default-avatar.jpg')), filename: 'default-avatar.jpg', content_type: 'image/jpg')
+    self.avatar.attach(io: File.open(Rails.root.join('app', 'assets', 'images', 'avatars', random_avatar)), filename: random_avatar, content_type: 'image/jpg')
+  end
+
+  def random_avatar
```
Comment (BrianBorge) [inline app/models/user.rb:22]: nice

File: Gemfile (hunk 1 above)
Comment (alexandraborge) [reply Gemfile:18]: true dat

File: Gemfile (hunk 2 above)
Comment (alexandraborge) [reply Gemfile:26]: nope, I will

File: app/views/users/edit.html.erb (hunk 3 above)
Comment (alexandraborge) [reply app/views/users/edit.html.erb:15]: I figured, I wanted to remember to add it back once the active storage issue is figured out

**Pull request merged**